lib/bambora/v1/profile_resource.rb

```diff
@@ -58,6 +58,48 @@ def create(card_data)
         client.post(path: sub_path, body: card_data, api_key: api_key)
       end
 
+      ##
+      # Get a Bambora payment profile given a customer code.
+      #
+      # @example
+      #
+      #   client = Bambora::Rest::JSONClient(base_url: '...', api_key: '...', merchant_id: '...')
+      #   profiles = Bambora::V1::ProfileResource(client: client)
+      #   customer_code = '02355E2e58Bf488EAB4EaFAD7083dB6A'
+      #
+      #   profiles.get(customer_code: customer_code)
+      #   # => {
+      #   #      :code => 1,
```

Comment:
Curious, why the hash rocket syntax here if the keys are all symbolified?

Reply:
@cassidycodes the reason for that is to emulate as close as possible what one would see in the REPL when running this.